testtools/testcase.py

```diff
@@ -276,6 +276,10 @@ def __eq__(self, other):
             return False
         return self.__dict__ == other.__dict__
 
+    def __hash__(self):
+        hashfn = getattr(unittest.TestCase, '__hash__', id)
+        return hashfn(self)
```

Comment:
This seems basically reasonable, but it's more complicated than necessary now that testtools no longer supports Python 2.  Could you make this be just:

```python
    __hash__ = unittest.TestCase.__hash__
```